app/core/constants.js

```diff
@@ -50,6 +51,7 @@ const LICENSE_PRESETS = {
 }
 
 const FREE_COURSE_IDS = [
+  '65f32b6c87c07dbeb5ba1936', // Junior
```

Comment:
are we sure we will free Junior campaigns ?

Reply:
For now, we need to have some introduction for elementary school students. We probably will just have either the first part of it be free or do a second Junior campaign for all the paid stuff.